app/components/post_draft/draft_handler/index.ts

```diff
@@ -42,4 +43,4 @@ const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
     };
 });
 
-export default withDatabase(enhanced(DraftHandler));
```

Comment:
Every time the PostDraft changes because of the update of the postInputTop, the whole draft re-renders, just because its parent re-rendered. Memoizing this takes away around 20ms.